jinja/index.html

```diff
@@ -119,6 +119,11 @@ <h5 class="white-text" style="font-family: 'Roboto', sans-serif !important;">Pen
                         <img class="img-responsive" src="img/aqt_big_transparent.png" style="max-width: 125px;"></img>
                     </a>
                 </td>
+                <td class="col-12 col-md-12 col-lg-2 mb-5">
+                    <a href="https://pennylane-ionq.readthedocs.io">
+                        <img class="img-responsive" src="img/ionq_logo.png" style="max-width: 150px;"></img>
+                    </a>
+                </td>
```

Comment:
Unfortunately, this is one too many logos, causing an overflow. Any thoughts 🤔

Reply:
Let's put TF, PyTorch (+Jax + Numpy) on a 3rd row

Reply:
Added a third row, looks much better :+1: